### PR TITLE
Rename slot size translations

### PR DESCRIPTION
**Pull request by joonvena**

### CODE REVIEW

File: locale/sv/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -994,7 +994,7 @@ msgstr "Minsta bokningstid"
 msgid "Maximum reservation time"
 msgstr "Längsta bokningstid"
 
-msgid "Slot size for reservation time"
+msgid "Slot size for reservation time in calendar view"
 msgstr "Spårstorlek för bokningstid"
```
Comment (jopesy) [inline locale/sv/LC_MESSAGES/django.po]: "Spårstorlek för bokningstid i kalendervy"

**Review by jopesy — DISMISSED** [review]

**Review by jopesy — COMMENTED** [review]
Looks good. I was thinking we'll run into migration conflicts if we want to merge changes from Helsinki but since this is just a change to the verbose name of a field, it shouldn't be too hard to fix any conflicts at that point.

Just update the Swedish translations, and it's good to go.

File: locale/sv/LC_MESSAGES/django.po (hunk 1 above)
Comment (joonvena) [reply locale/sv/LC_MESSAGES/django.po]: Added

**Review by jopesy — APPROVED** [review]
👍

**Pull request merged**